styles/go.dev/catppuccin.user.css

```diff
@@ -263,8 +263,8 @@
       filter: none;
     }
 
-    .SiteBreadcrumb li.BreadcrumbNav-li::after {
-      @svg: escape('<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><mask id="a" mask-type="alpha" maskUnits="userSpaceOnUse" x="5" y="2" width="7" height="12"><path d="m5.06 12.393.94.94L11.333 8 6 2.667l-.94.94L9.447 8" fill="@{text}"/></mask><g mask="url(#a)"><rect width="16" height="16" fill="#f0f1f2"/></g></svg>');
+    .SiteBreadcrumb .BreadcrumbNav-li:after {
+      @svg: escape('<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><mask id="a" mask-type="alpha" maskUnits="userSpaceOnUse" x="5" y="2" width="7" height="12"><path d="m5.06 12.393.94.94L11.333 8 6 2.667l-.94.94L9.447 8" fill="#fff"/></mask><g mask="url(#a)"><rect width="16" height="16" fill="@{text}"/></g></svg>');
```

Comment:
Ideally we shouldn't have any `#fff` here right? Does that `fill` attribute need to exist?

Reply:
It does. Removing it or setting it to none results in the svg not being visible.

Reply:
I mean I think you can remove the mask entirely?

```xml
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m5.06 12.393.94.94L11.333 8 6 2.667l-.94.94L9.447 8" fill="@{text}"/></svg>
```